setup.py

```diff
@@ -133,6 +135,7 @@ def get_version(version_file):
         "console_scripts": [
             "pe-reports = pe_reports.report_generator:main",
             "pe-mailer = pe_mailer.email_reports:main",
+            "pe-source = pe_source.pe_scripts:main",
```

Comment:
Keep this in alphabetical order:
```suggestion
            "pe-mailer = pe_mailer.email_reports:main",
            "pe-reports = pe_reports.report_generator:main",
            "pe-source = pe_source.pe_scripts:main",
```